.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,40 +1,62 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Report a bug with the WordPress block editor or Gutenberg plugin
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+<!--
+Please fill out ALL required sections. Bug reports with missing information will be
+closed.
 
-**To reproduce**
-Steps to reproduce the behavior:
+Before submitting a bug report:
+
+- Check if the bug has already been fixed by updating WordPress and/or Gutenberg.
+- Check if the bug is caused by a plugin by deactivating all plugins except Gutenberg.
+- Check if the bug is caused by a theme by activating a default theme e.g. Twenty Twenty.
+
+If this is a security issue, please report it in HackerOne instead:
+https://hackerone.com/wordpress
+-->
+
+## Description
+<!-- Please write a brief description of the bug. -->
+
+## Step-by-step reproduction instructions
+<!--
+Please list the steps needed to reproduce the bug. For example:
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
-
-**Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
-
-**Editor version (please complete the following information):**
-- WordPress version: [e.g: 5.3.2]
-- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"]
-- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]
-
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
-
-**Additional context**
-- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.
+2. Click on '...'
+3. Scroll down to '...'
+-->
+
+## Expected behaviour
+<!-- Please describe what you expected to happen. -->
+
+## Actual behaviour
+<!-- Please describe what actually happened. -->
+
+## Screenshots or screen recording (optional)
+<!--
+If possible, please upload a screenshot or screen recording which demonstrates
+the bug. You can use LIEcap to create a GIF screen recording:
+https://www.cockos.com/licecap/
+-->
+
+## Plugin code (optional)
+<!--
+If this bug is to do with a developer API, please share a code snippet or plugin
+code that demonstrates the issue. You can use GitHub Gist to share multiple code
+files:
+https://gist.github.com
+-->
```

Comment:
```suggestion
## Code snippet (optional)
<!--
If this bug is to related to a developer API, please share a code snippet that 
demonstrates the issue.

For small snippets paste it directly here, or you can use GitHub Gist to share
multiple code files: https://gist.github.com

Please ensure the shared code can be used by a developer to reproduce the 
issue—ideally it can be copied into a local development environment or executed
in a browser console to help debug the issue.
-->
```

Reply:
@noisysocks How does that sound?

Reply:
Very good!